Dwolla.Client/Models/Responses/BusinessClassification.cs

```diff
@@ -4,21 +4,23 @@
 
 namespace Dwolla.Client.Models.Responses
 {
-    public class BusinessClassification
+    public class BusinessClassification : BaseResponse
     {
-        [JsonProperty(PropertyName = "_links")]
-        public Dictionary<string, Link> Links { get; set; }
-
         [JsonProperty(PropertyName = "_embedded")]
         public IndustryClassificationsEmbed Embedded { get; set; }
 
         public string Id { get; set; }
         public string Name { get; set; }
     }
 
-    public class IndustryClassificationsEmbed
+    public class IndustryClassificationsEmbed : Embed<IndustryClassification>
     {
         [JsonProperty(PropertyName = "industry-classifications")]
         public List<IndustryClassification> IndustryClassifications { get; set; }
+
+        public override List<IndustryClassification> Results()
+        {
+            return IndustryClassifications;
+        }
```

Comment:
This (and the other Results) can be expression-bodied,
`public override List<IndustryClassification> Results() => IndustryClassifications;`